data/qrySumStatsAllCharterDistAllYears_v2.csv

```diff
@@ -2469,28 +2469,28 @@
-"SER-Ni+�os Charter School",1999,1,237.00,219.00,221.00,221.00,8.00
-"SER-Ni+�os Charter School",2000,1,258.00,232.00,243.00,243.00,11.00
-"SER-Ni+�os Charter School",2001,1,330.00,277.00,308.00,308.00,14.00
-"SER-Ni+�os Charter School",2002,1,370.00,266.00,346.00,347.00,17.00
-"SER-Ni+�os Charter School",2003,1,376.00,340.00,363.00,363.00,18.00
-"SER-Ni+�os Charter School",2004,1,392.00,369.00,374.00,374.00,22.00
-"SER-Ni+�os Charter School",2005,1,537.00,499.00,504.00,504.00,24.00
-"SER-Ni+�os Charter School",2006,1,507.00,473.00,456.00,456.00,25.00
-"SER-Ni+�os Charter School",2007,1,563.00,531.00,498.00,498.00,24.00
-"SER-Ni+�os Charter School",2008,2,587.00,547.00,520.00,520.00,31.00
-"SER-Ni+�os Charter School",2009,2,607.00,474.00,554.00,554.00,24.00
-"SER-Ni+�os Charter School",2010,2,622.00,522.00,531.00,531.00,24.00
-"SER-Ni+�os Charter School",2011,2,615.00,556.00,522.00,522.00,18.00
-"SER-Ni+�os Charter School",2012,3,831.00,788.00,734.00,781.00,36.00
-"SER-Ni+�os Charter School",2013,3,954.00,917.00,847.00,886.00,38.00
-"SER-Ni+�os Charter School",2014,3,1013.00,984.00,888.00,946.00,41.00
-"SER-Ni+�os Charter School",2015,3,1031.00,996.00,936.00,961.00,41.00
-"SER-Ni+�os Charter School",2016,3,1087.00,1046.00,964.00,1017.00,41.00
-"SER-Ni+�os Charter School",2017,3,1104.00,1063.00,1004.00,1005.00,42.00
-"SER-Ni+�os Charter School",2018,3,1107.00,1059.00,957.00,994.00,53.00
+"SER-Niños Charter School",1997,1,150.00,144.00,123.00,122.00,5.00
+"SER-Niños Charter School",1998,1,219.00,206.00,192.00,192.00,9.00
+"SER-Niños Charter School",1999,1,237.00,219.00,221.00,221.00,8.00
+"SER-Niños Charter School",2000,1,258.00,232.00,243.00,243.00,11.00
+"SER-Niños Charter School",2001,1,330.00,277.00,308.00,308.00,14.00
+"SER-Niños Charter School",2002,1,370.00,266.00,346.00,347.00,17.00
+"SER-Niños Charter School",2003,1,376.00,340.00,363.00,363.00,18.00
+"SER-Niños Charter School",2004,1,392.00,369.00,374.00,374.00,22.00
+"SER-Niños Charter School",2005,1,537.00,499.00,504.00,504.00,24.00
+"SER-Niños Charter School",2006,1,507.00,473.00,456.00,456.00,25.00
+"SER-Niños Charter School",2007,1,563.00,531.00,498.00,498.00,24.00
+"SER-Niños Charter School",2008,2,587.00,547.00,520.00,520.00,31.00
+"SER-Niños Charter School",2009,2,607.00,474.00,554.00,554.00,24.00
+"SER-Niños Charter School",2010,2,622.00,522.00,531.00,531.00,24.00
+"SER-Niños Charter School",2011,2,615.00,556.00,522.00,522.00,18.00
+"SER-Niños Charter School",2012,3,831.00,788.00,734.00,781.00,36.00
+"SER-Niños Charter School",2013,3,954.00,917.00,847.00,886.00,38.00
+"SER-Niños Charter School",2014,3,1013.00,984.00,888.00,946.00,41.00
+"SER-Niños Charter School",2015,3,1031.00,996.00,936.00,961.00,41.00
+"SER-Niños Charter School",2016,3,1087.00,1046.00,964.00,1017.00,41.00
+"SER-Niños Charter School",2017,3,1104.00,1063.00,1004.00,1005.00,42.00
+"SER-Niños Charter School",2018,3,1107.00,1059.00,957.00,994.00,53.00

```

Comment:
This is the result of an Excel for Mac bug that we'll have to watch out for if and when there are data updates.  Basically if you open a CSV with non-ASCII characters in Excel for Mac it displays fine, but then on saving they get mangled.  It's not too hard to fix this by opening the file in a decent text editor, which is what I did here, but the better course of action (NOTE TO SELF) is never to save from Excel for Mac in the first place.

Reply:
This is good to know